lib/assets/javascripts/turbograft/turbohead.coffee

```diff
@@ -14,6 +22,10 @@ class window.TurboHead
       .filter(attributeMatches('nodeName', 'LINK'))
       .filter(noAttributeMatchesIn('href', @activeAssets))
 
+  @reset: ->
```

Comment:
We could hide it inside of an object called `_testAPI` or something. We use that in the theme editor and I like it